perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -155,19 +156,27 @@ sub construct_bin_request_form {
                 tags => { toggle => "form-quantity-$id-row" },
             };
             $name = ''; # Only on first container
-            push @$field_list, "quantity-$id" => {
-                type => 'Select',
-                label => 'Quantity',
-                tags => {
-                    hint => "You can request a maximum of " . NUMWORDS($max) . " containers",
-                    initial_hidden => 1,
-                },
-                options => [
-                    { value => "", label => '-' },
-                    map { { value => $_, label => $_ } } (1..$max),
-                ],
-                required_when => { "container-$id" => 1 },
-            };
+            if ($max == 1) {
+                push @$field_list, "quantity-$id" => {
```

Comment:
This will then let through a bigger number, because Hidden field isn't validated against options, so I think it needs a check (or hard-coding elsewhere later on).